usage-examples/test-execution.md

```diff
@@ -6,6 +6,8 @@ Using Eiffel events to describe the execution of a test activity can be done in
 
 The Eiffel protocol provides several events to use within a test activity. This example shows how to make use of most of the Eiffel events related to test activities, with the purpose to present how they all are related.
 
+To understand the example provided here, it is important to explain a few concepts. While Eiffel does not make any assumptions about the underlying infrastructure and/or testing methodology, it does encourage separation of concerns. In this view, it is not the responsibility of the activity orchestrator (e.g. the CI server) to determine the contents of the test scope, nor is it the responsibility of the test executor. Instead, the [EiffelTestExecutionRecipeCollectionCreatedEvent](../eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md) makes it possible for a third actor to determine the test scope, which is then handed off to a test executor. And a forth actor is used to provision and deploy a test environment, based on the environment needs described in the [EiffelTestExecutionRecipeCollectionCreatedEvent](../eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md), for the test to be executed towards, which is also then handed off to the test executor.
```

Comment:
Typo: forth

I'm having trouble with the syntax of the last sentence. How about "Then a fourth actor may be used to set up and provide a test environment, based on the needs described in the [EiffelTestExecutionRecipeCollectionCreatedEvent](../eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md), which the test executor may use to run the tests.